web/web.go

```diff
@@ -71,6 +71,17 @@ import (
 
 var localhostRepresentations = []string{"127.0.0.1", "localhost"}
 
+// SecureHeaders adds secure headers to the responses
```

Comment:
It's secureHeadersMiddleware.

Comments should end with a full stop.

Reply:
> All top-level, exported names should have doc comments, as should non-trivial unexported type or function declarations. See https://golang.org/doc/effective_go.html#commentary for more information about commentary conventions.

`secureHeadersMiddleware` is trivial so should not need a comment.